DESCRIPTION

```diff
@@ -39,6 +39,7 @@ Imports:
     utils
 Suggests:
     knitr (>= 1.42),
+    random.cdisc.data (>= 0.2.1),
```

Comment:
Technically, this is an optional dependency as users only see/use it in the documentation (in `@source` roxygen2 tags)